### DISCUSSION TITLE
How to updated HTML in please wait js

### DISCUSSION THREAD

**Discussion by topstarwebking** [community/community#20845] [Other Feature Feedback, Questions, & Ideas] [answered]
I can't updated HTML content in please-wait.js
If anyone know this , please leave comment kindly.
Thanks in advance.

**Comment by Wuffle1125** [comment, marked as answer]
Hello @snowflake-dev918 

I have some experiences in please-wait.js
you can use updatedLoadingHtml() function.
You can pass HTML content using first parameter.

Hope this will be help.

**Comment by devlancer412** [comment]
Hello @snowflake-dev918

you can use updatedLoadingHtml() function.
You can pass HTML content using first parameter.
